app/Models/Mship/Concerns/HasQualifications.php

```diff
@@ -61,7 +61,7 @@ public function addQualification(Qualification $qualification)
      * @param int $atcRating The VATSIM ATC rating
```

Comment:
Docstrings don't reflect the optional param

Reply:
43e8d4fe